Index.html

```diff
@@ -68,6 +68,243 @@ <h2>I’m Hussen Glad to see you!</h2>
+                      <img src="img/work-img/Counter.png" />
+                     </li>
+                     <li class="frames-list">
+                      <h2>Back End Dev</h2>
+                     </li>
+                     <li class="frames-list">
+                      <img src="img/work-img/Counter.png" />
+                     </li>
+                     <li class="frames-list">
+                      <h2>2015</h2>
+                     </li>
+                 </ul>
+                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 
+                 <ul class="tag-block">
+                  <li class="tag-list">
+                     <h2>html</h2>
+                    </li>
+                     <li class="tag-list">
+                      <h2>css</h2>
+                     </li>
+                     <li class="tag-list">
+                      <h2>javaScript</h2>
+                     </li>
+                 </ul>
+                  <div class="button-acction">
+                      <button class="button-acction-button" type="button"> See Project</button>
+                 </div>
+              </div>
+              </div>
+              </div>
+            </div>
+          </div>
+      </section>
+
+      <section class="my-self-container">
+        <div class="top">
+          <div class="text-p">
+            <h2>About
+                Myself</h2>
+          </div>
+          <div class="text-s">
+            <p>
+                Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="lets-connect-p">
+            <a href="#"><b>LET’S CONNECT</b></a>
+          </div>
+
+          <ul class="social-media-icons-p">
+            <li class="list-p">
+                <a href="https://github.com/HussenYesufAlli">
+                  <img src="img/g.png" alt="github" />
+                </a>
+              </li>
+            
+            <li class="list-p">
+              <a href="https://Linkedin.com/">
+                <img src="img/Linkedin.png" alt="linkedin" />
+              </a>
+            </li>
+            <li class="list-p">
+                <a href=""> <img src="img/f.png" alt="f" /> </a>
+              </li>
+            <li class="list-p">
+                <a href="https://twitter.com/">
+                  <img src="img/t.png" alt="tiwetter" />
+                </a>
+              </li>
+            <li class="list">
+              <a href=""> <img src="img/m.png" alt="medium" /> </a>
+            </li>
+            
+          </ul>
+          <div class="button-my">
+            <button class="my-button" type="button"> Get my resume</button>
+       </div>
+        </div>
+      </section>
+
```

Comment:
- [ ] I kindly suggest that you add your `technologies categories` after your `Get my resume` button

<img width="351" alt="image" src="https://user-images.githubusercontent.com/52934128/221383772-838478d3-3d51-4ec9-93fa-e66bce14f245.png">